app/helpers/patient_helper.rb

```diff
@@ -1,384 +1,16 @@
 # frozen_string_literal: true
 
 # Helper methods for the patient model
-module PatientHelper # rubocop:todo Metrics/ModuleLength
+module PatientHelper
   # This list contains all of the same states listed in app/javascript/components/data.js
   def state_names
-    {
-      'alabama' => 'Alabama',
-      'alaska' => 'Alaska',
-      'americansamoa' => 'American Samoa',
-      'arizona' => 'Arizona',
-      'arkansas' => 'Arkansas',
-      'california' => 'California',
-      'colorado' => 'Colorado',
-      'connecticut' => 'Connecticut',
-      'delaware' => 'Delaware',
-      'districtofcolumbia' => 'District of Columbia',
-      'federatedstatesofmicronesia' => 'Federated States of Micronesia',
-      'florida' => 'Florida',
-      'georgia' => 'Georgia',
-      'guam' => 'Guam',
-      'hawaii' => 'Hawaii',
-      'idaho' => 'Idaho',
-      'illinois' => 'Illinois',
-      'indiana' => 'Indiana',
-      'iowa' => 'Iowa',
-      'kansas' => 'Kansas',
-      'kentucky' => 'Kentucky',
-      'louisiana' => 'Louisiana',
-      'maine' => 'Maine',
-      'marshallislands' => 'Marshall Islands',
-      'maryland' => 'Maryland',
-      'massachusetts' => 'Massachusetts',
-      'michigan' => 'Michigan',
-      'minnesota' => 'Minnesota',
-      'mississippi' => 'Mississippi',
-      'missouri' => 'Missouri',
-      'montana' => 'Montana',
-      'nebraska' => 'Nebraska',
-      'nevada' => 'Nevada',
-      'newhampshire' => 'New Hampshire',
-      'newjersey' => 'New Jersey',
-      'newmexico' => 'New Mexico',
-      'newyork' => 'New York',
-      'northcarolina' => 'North Carolina',
-      'northdakota' => 'North Dakota',
-      'northernmarianaislands' => 'Northern Mariana Islands',
-      'ohio' => 'Ohio',
-      'oklahoma' => 'Oklahoma',
-      'oregon' => 'Oregon',
-      'palau' => 'Palau',
-      'pennsylvania' => 'Pennsylvania',
-      'puertorico' => 'Puerto Rico',
-      'rhodeisland' => 'Rhode Island',
-      'southcarolina' => 'South Carolina',
-      'southdakota' => 'South Dakota',
-      'tennessee' => 'Tennessee',
-      'texas' => 'Texas',
-      'utah' => 'Utah',
-      'vermont' => 'Vermont',
-      'virginislands' => 'Virgin Islands',
-      'virginia' => 'Virginia',
-      'washington' => 'Washington',
-      'westvirginia' => 'West Virginia',
-      'wisconsin' => 'Wisconsin',
-      'wyoming' => 'Wyoming'
-    }
+    @state_names ||= YAML.safe_load(File.read('lib/assets/state_names.yml'), [Symbol])
```

Comment:
Why do a `||=` here? I'd imagine we always want to overwrite.

Reply:
Generally I don't think we should expect the contents of these files to change dynamically as the app runs. Also, it should be less of a performance hit to not load in the file every time and re-use if it is already set.

Reply:
Gotcha, that makes sense.

Reply:
I'm wary of loading files like this outside of config&initializers - could we just put this in config like we do for the sara config?

Reply:
@holmesie Just pushed this change to load in initializers